x-pack/solutions/security/packages/security-ai-prompts/src/mock_prompts.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Prompt } from './types';
+
+export const promptGroupId = {
+  attackDiscovery: 'attackDiscovery',
+  aiAssistant: 'aiAssistant',
+};
+
+export const promptDictionary = {
+  systemPrompt: `systemPrompt`,
+  userPrompt: `userPrompt`,
+  attackDiscoveryDefault: `default`,
+  attackDiscoveryRefine: `refine`,
+};
+
+export const localPrompts: Prompt[] = [
+  {
+    promptId: promptDictionary.systemPrompt,
+    promptGroupId: promptGroupId.aiAssistant,
+    provider: 'openai',
+    prompt: {
+      default: 'provider:openai default system prompt',
+    },
+  },
+  {
+    promptId: promptDictionary.systemPrompt,
+    promptGroupId: promptGroupId.aiAssistant,
+    prompt: {
+      default: 'provider:default default system prompt',
```

Comment:
Consider omitting `provider:default`, i.e.

```typescript
default: 'default system prompt',
```

for consistency with the convention used when `model` is `undefined`.

For example, in this prompt,`model` is `undefined`, yet the `prompt` does not include `model:default`.